src/app/main.cpp

```diff
@@ -124,12 +124,10 @@ int main( int argc, char **argv )
   app.setThemeName( settings.value( "/Themes", "default" ).toString() );
 
 #if defined( Q_OS_WIN )
-  qputenv( "GDAL_DATA", QDir::toNativeSeparators( app.applicationDirPath() + "/gdal" ).toLocal8Bit() );
-  qputenv( "PROJ_LIB", QDir::toNativeSeparators( app.applicationDirPath() + "/proj" ).toLocal8Bit() );
-  app.setPrefixPath( app.applicationDirPath() + "/qgis", true );
-#else
-  app.setPrefixPath( CMAKE_INSTALL_PREFIX, true );
+  qputenv( "GDAL_DATA", QDir::toNativeSeparators( app.applicationDirPath() + "/../share/gdal" ).toLocal8Bit() );
+  qputenv( "PROJ_LIB", QDir::toNativeSeparators( app.applicationDirPath() + "/../share/proj" ).toLocal8Bit() );
 #endif
+  app.setPrefixPath( app.applicationDirPath() + "/..", true );
```

Comment:
@m-kuhn , is that breaking "normal" linux builds? (I.e. building from QtCreator)